src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/PackageRestoreUnconfiguredInputDataSource.cs

```diff
@@ -38,37 +38,26 @@ protected override IDisposable LinkExternalInput(ITargetBlock<RestoreInfo> targe
             // (via ProjectRestoreUpdate) and combine it into a single IVsProjectRestoreInfo2 instance and publish that. When a change is 
             // made to a configuration, such as adding a PackageReference, we should react to it and push a new version of our output. If the 
             // active configuration changes, we should react to it, and publish data from the new set of implicitly active configurations.
```

Comment:
Are the comments here and above still applicable due to the changes in which dataflow block is used? If so, it sounds like it should be above the method itself, not inside of it.

Reply:
Yes the comment is still valid. It's explaining what's happening, rather than how it happens. We just change the how here.